SUIApp/SUIApp/PurchaseModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  PurchaseModel.swift
+//  SUIApp
+//
+//  Created by Vladka on 22.01.2023.
+//
+
+import Foundation
+
+/// View Model
```

Comment:
```suggestion
/// View Model экрана выбора тарифов
```
Я бы наверно чуть-чуть конкретизировал чья модель именно